ci/rpm_build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Script for buiding RPMs in a chroot
+# If the STAGE_NAME environment variable is present the script will
+# attempt to parse it to determine what distribution to build for.
+#
+# For manual testing, you can set the environment variables CHROOT_NAME
+# and TARGET can be set.
+#
+# Default is to build for CentOS 7.
+
+set -ex
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/parse_ci_envs.sh"
+if [ -e "${ci_envs}" ]; then
+  # shellcheck disable=SC1091
```

Comment:
As I recall, that behavior varied with the version of shellcheck.  Only the shellcheck disable line above made all versions of shellcheck happy between Centos 7, Ubuntu, and  Sles-12
Which is why I want to try a future PR to use fedora:latest for the code review stage instead of CentOS.

Reply:
@brianjmurrell is right, it would be much better to let ShellCheck follow this.  Hopefully this can be added once landed.

Reply:
> If I put the shellcheck source line in, it only works on the newer versions of shellcheck, and it causes the other versions of shell check to flag it with an unrecognized shellcheck command.

But we only use one version of ShellCheck at any point in time.

> The shellcheck disable SC1091 is the only pragma that works for all the distros

What do you mean "all the distros"?  We only run checkpatch on a single distro -- EL7.  Although arguably that should be Fedora since it has much newer linting tools including a much newer ShellCheck.  But that's a different PR.

> that we we were using 6 months ago, and I will not be in able to verify that our current and planned set of distros provide a new enough shellcheck package for a while.

Again, it doesn't need to work on every distro, just the one we actually run linting on which is currently EL7.

That should reduce the complexity of being able to fix this one without **disable**, yes?

Reply:
Confirmed "# shellcheck source pathname" will not work in the ShellCheck package installed by CentOS.
Don't want to make any checkpatch changes in this PR.

"Couldn't parse this shellcheck annotation. [SC1073]\n(lint) Expected \"disable\". Fix any mentioned problems and try again. [SC1072]"

Reply:
Seems it was added in 0.4.0 and EL7 has 0.3.8.  We really need to get all of this stuff pushed up to running on Fedora instead of EL7.